.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I'd like to know how to use it, could you do a short demo on the frontend meeting we have on Thursday ?
I think it should be here:
![image](https://user-images.githubusercontent.com/10500650/106576646-b2140900-653d-11eb-8baf-d6d032503c01.png)
but I don't know all the features it has

Reply:
Sure, sounds good